core/model/config.go

```diff
@@ -18,23 +18,42 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/rokwire/logging-library-go/errors"
 	"github.com/rokwire/logging-library-go/logutils"
 )
 
 const (
-	//TypeGlobalConfig ...
-	TypeGlobalConfig logutils.MessageDataType = "global config"
+	// TypeConfig configs type
+	TypeConfig logutils.MessageDataType = "config"
+	// TypeEnvConfigData env configs type
+	TypeEnvConfigData logutils.MessageDataType = "env config data"
 	//TypeOrganizationConfig ...
 	TypeOrganizationConfig logutils.MessageDataType = "org config"
+
+	// ConfigIDEnv is the Config ID for EnvConfigData
+	ConfigIDEnv string = "env"
 )
 
-// GlobalConfig represents global config for the system
-type GlobalConfig struct {
-	Setting string
+// Config contains generic configs
+type Config struct {
```

Comment:
We should probably apply the latest config chanes described here: https://github.com/rokwire/groups-building-block/issues/346